src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -1029,7 +985,10 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
   absl::flat_hash_map<VkCommandBuffer, VkDevice> command_buffer_to_device_;
 
   absl::flat_hash_map<VkCommandBuffer, CommandBufferState> command_buffer_to_state_;
-  absl::flat_hash_map<VkQueue, std::vector<QueueSubmission>> queue_to_submissions_;
+  absl::flat_hash_map<VkQueue,
+                      std::priority_queue<QueueSubmission, std::vector<QueueSubmission>,
+                                          std::function<bool(QueueSubmission, QueueSubmission)>>>
+      submissions_queue_;
```

Comment:
Do you want the singular here? These are multiple priority queues. Or consider expanding the name to something like `queue_to_submission_priority_queue_`.

Reply:
Renamed it as you suggested.